packages/peregrine/lib/Apollo/policies/__tests__/index.spec.js

```diff
@@ -0,0 +1,359 @@
+import typePolicies from '../index';
+
+const readField = jest.fn((name, object) => {
+    if (typeof object[name] === Array) {
+        return object[name].join();
+    }
+    return object[name];
+});
+
+const mergeObjects = jest.fn((a, b) => {
+    return Object.assign({}, a, b);
+});
+
+const address = {
+    city: 'Austin',
+    firsname: 'Veronica',
+    lastname: 'Costello',
+    street: ['11501 Domain Dr', 'Suite 3'],
+    postcode: '78758',
+    telephone: '555-123-4567'
+};
+
+const address2 = {
+    city: 'Los Angeles',
+    firsname: 'Veronica',
+    lastname: 'Costello',
+    street: ['3640 Holdrege Ave'],
+    postcode: '90016',
+    telephone: '555-456-7890'
+};
+
+test('provides an object with the correct shape', () => {
+    expect(typePolicies).toMatchSnapshot();
```

Comment:
Would like to bring this test up at dev time; I'm unsure of its value, but do want to discuss our plans to leverage unit tests more to detect breakages. I don't know if type policy configuration is API, but it certainly could have a negative impact on the cache integrity if something is changed. Not blocking approval, just wanted to discuss.